Dockerfile

```
@@ -5,19 +5,15 @@
 FROM golang:1.18.4 AS builder
 
 WORKDIR /workspace
-COPY go.mod go.mod
-COPY go.sum go.sum
-# cache deps before building and copying source so that we don't need to re-download as much
-# and so that source changes don't invalidate our downloaded layer
+COPY . .
 RUN go mod download
 
-# Copy the go source
-COPY cmd/ cmd/
-COPY pkg/ pkg/
-COPY internal/ internal/
+# Get version
+RUN hack/get-build.sh > /tmp/build-flags
 
 # Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o service-account-issuer-discovery cmd/service-account-issuer-discovery/main.go
+WORKDIR cmd/service-account-issuer-discovery
```

Please use an absolute path.

```suggestion
WORKDIR /workspace/cmd/service-account-issuer-discovery
```